LeetCode 153 - Find Minimum in Rotated Sorted Array.java

```diff
@@ -0,0 +1,28 @@
+// Time Complexity: Since we use binary search methods here, 
+// so the time complexity will be O(logn).
+/* --------------------------------------------------------*/
+// Space Complexity: the space complexity will be O(1).
+/* --------------------------------------------------------*/
+// I am still confused about the how to consider the boundaries of these kinds 
+// of problems, so that we will not miss any ones?
+
```

Comment:
try to visualise the problem and think of how the mid plays along 
Take happy case and try to tweak around with it. You'll get an idea of indexes are moving and you'll understand the edge cases